geode-core/src/distributedTest/java/org/apache/geode/internal/cache/SingleHopStatsDUnitTest.java

```diff
@@ -331,12 +337,12 @@ private void createdColocatedPRData(GemFireCacheImpl cache) {
 
       customerRegion.put(custid, customer);
       for (int j = 1; j <= 10; j++) {
-        int oid = (i * 10) + j;
+        int oid = i * 10 + j;
```

Comment:
You're going to make me go look up the Java operator precedence to make sure this is doing what I really think it is?

Reply:
I just cheat and let the IDE tell me the precedence! Multiplication happens before addition.